apps/sites/static/pages/js/user_story_feedback.js

```
@@ -43,6 +44,43 @@
+    const borderBottom = parseFloat(computed.borderBottomWidth) || 0;
+    const nextHeight = field.scrollHeight + borderTop + borderBottom;
+
+    if (!force && nextHeight === 0) {
+      return;
+    }
+
+    field.style.height = `${nextHeight}px`;
+  };
+
+  const resizeFeedbackTextareas = options => {
+    feedbackTextareas.forEach(textarea => resizeTextarea(textarea, options));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)